samtranslator/model/eventsources/push.py

```diff
@@ -484,6 +484,23 @@ def to_cloudformation(self, **kwargs):
         return resources
 
 
+class AlexaSmartHome(PushEventSource):
```

Comment:
Move this to `alexa-smart-home.py` (or `alexasmarthome.py` - need to decided on naming convention)